### ISSUE TITLE
Graphs / outputs related to this project 

### ISSUE THREAD

**Issue by vivek2319** [vivek2319/Predicting-US-Census-Income#1]
![45](https://user-images.githubusercontent.com/21111859/33800200-7cacb6a0-dd09-11e7-9eaf-fb3288fce496.png)
![89](https://user-images.githubusercontent.com/21111859/33800201-7eab58da-dd09-11e7-9768-e540fb9c05c5.png)
![95](https://user-images.githubusercontent.com/21111859/33801155-a113bf14-dd1f-11e7-9caf-8498a23f1dc9.png)

![100](https://user-images.githubusercontent.com/21111859/33800208-95cfa7a0-dd09-11e7-9064-0941cfbfa2ec.png)
![105](https://user-images.githubusercontent.com/21111859/33800210-9a02a2d2-dd09-11e7-9d25-0e4b456406f2.png)

![110](https://user-images.githubusercontent.com/21111859/33800205-86484eae-dd09-11e7-9853-3b6777cb6f2f.png)
![115](https://user-images.githubusercontent.com/21111859/33800207-8b379b4a-dd09-11e7-88e8-1ba69b7213c7.png)

![120](https://user-images.githubusercontent.com/21111859/33800215-ab6f1618-dd09-11e7-8f78-7ffc9c1b44a0.png)
![125](https://user-images.githubusercontent.com/21111859/33800217-aedcc386-dd09-11e7-9994-5b34fb60e564.png)
![lda](https://user-images.githubusercontent.com/21111859/33800223-bca1c34a-dd09-11e7-9db9-fe30b21956f4.png)
![knn](https://user-images.githubusercontent.com/21111859/33800224-c6ea0330-dd09-11e7-953f-134653496d13.png)
![linearregression](https://user-images.githubusercontent.com/21111859/33800225-ccb031fe-dd09-11e7-9580-c385bc384993.png)
![decisiontreemodel](https://user-images.githubusercontent.com/21111859/33800228-e1a51fde-dd09-11e7-88ce-e1a8967b4664.png)
![kmeans accuracy](https://user-images.githubusercontent.com/21111859/33800229-e1b8bcce-dd09-11e7-8139-7eb5072f6943.png)
![naivebayes](https://user-images.githubusercontent.com/21111859/33800230-e1ce14de-dd09-11e7-886f-6e34a931f12a.png)
![neuralnetwork](https://user-images.githubusercontent.com/21111859/33800231-e1e05fc2-dd09-11e7-8807-4126f78ac133.png)
![svmaccuracy](https://user-images.githubusercontent.com/21111859/33800232-e1f29494-dd09-11e7-8517-928e6d778d1f.png)
![first learning rate](https://user-images.githubusercontent.com/21111859/33800238-ed0645a6-dd09-11e7-90d5-53032e45a39a.png)
![choosing best learning rate](https://user-images.githubusercontent.com/21111859/33800239-f1af0c3c-dd09-11e7-835d-cdcb5ad87e5d.png)
![best learning rate and n-estimators](https://user-images.githubusercontent.com/21111859/33800244-feb83066-dd09-11e7-8daa-435dd872160c.png)
![last log loss](https://user-images.githubusercontent.com/21111859/33800247-057ef358-dd0a-11e7-9ad6-8ba6d5ab3703.png)
![last-xgboost](https://user-images.githubusercontent.com/21111859/33800248-0a5d49e2-dd0a-11e7-8b65-31c2fa72c47b.png)
![picture1](https://user-images.githubusercontent.com/21111859/33803528-9ae0bc82-dd60-11e7-9563-4689561a37b5.png)

**Comment by vivek2319** [comment]
https://www.dropbox.com/s/aw8e819uop6ly92/Picture1.png?dl=0
